addons/account/wizard/setup_wizards.py

```diff
@@ -66,14 +66,27 @@ class SetupBarBankConfigWizard(models.TransientModel):
     _description = 'Bank setup manual config'
 
     res_partner_bank_id = fields.Many2one(comodel_name='res.partner.bank', ondelete='cascade', required=True)
-    new_journal_name = fields.Char(default=lambda self: self.linked_journal_id.name, inverse='set_linked_journal_id', required=True, help='Will be used to name the Journal related to this bank account')
-    linked_journal_id = fields.Many2one(string="Journal", comodel_name='account.journal', inverse='set_linked_journal_id', compute="_compute_linked_journal_id")
-    new_journal_code = fields.Char(string="Code", required=True, default=lambda self: self._onchange_new_journal_code())
-    num_journals_without_account = fields.Integer(default=lambda self: self._number_unlinked_journal())
+    new_journal_name = fields.Char(required=True, help='Will be used to name the Journal related to this bank account')
+    new_journal_code = fields.Char(string="Code", required=True, default=lambda self: self.env['account.journal'].get_next_bank_cash_default_code('bank', self.env.company.id))
+    journal_id = fields.Many2one('account.journal')
```

Comment:
Alas, those model changes aren't acceptable in stable :-(

Reply:
13.1 then? Or ugly context thing?